### PR TITLE
docs(CHANGELOG): add breaking change for NPM's peerDependencies

### LINKED ISSUE #5746: npm install complains about peer deps
I get the following warnings when I install the latest alpha version:
```shell
npm WARN EPEERINVALID angular2@2.0.0-alpha.50 requires a peer of es6-promise@^3.0.2 but none was installed.
npm WARN EPEERINVALID angular2@2.0.0-alpha.50 requires a peer of es6-shim@^0.33.3 but none was installed.
npm WARN EPEERINVALID angular2@2.0.0-alpha.50 requires a peer of reflect-metadata@0.1.2 but none was installed.
npm WARN EPEERINVALID angular2@2.0.0-alpha.50 requires a peer of rxjs@5.0.0-alpha.14 but none was installed.
npm WARN EPEERINVALID angular2@2.0.0-alpha.50 requires a peer of zone.js@0.5.8 but none was installed.
```

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Fixes #5746

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -64,6 +64,25 @@ Use `angular-polyfills.js` instead.
   - router is exported as `ng.router`
   - instrumentation is exported as `ng.instrumentation`
   - upgrade is exported as `ng.upgrade`
+  
+* rxjs, reflect-metadata, zone.js and es6-shims now must be specified as
```
Comment (alxhub) [inline CHANGELOG.md]: Nit: missing space in front of asterisk.

**Comment by IgorMinar** [comment]
Lgtm. As thisbbit picked up by the change of script? Please file a bug

File: CHANGELOG.md (hunk 1 above)
Comment (pkozlowski-opensource) [inline CHANGELOG.md]: @alxhub I can't see spaces in front of other `*` in our changelog (at least not when open in my editor)

**Comment by pkozlowski-opensource** [comment]
@IgorMinar yeh, this is due to the fact that commit was flagged as `build`. Will file an issue

File: CHANGELOG.md [hunk 2]
```diff
@@ -64,6 +64,25 @@ Use `angular-polyfills.js` instead.
...
+To migrate, please add the following into the "dependencies" section of your package.json:
+
+```
+"dependencies": {
+    ...
+
+    "es6-promise": "^3.0.2",
+    "es6-shim": "^0.33.3",
+    "reflect-metadata": "0.1.2",
+    "rxjs": "5.0.0-alpha.11",
```
Comment (meenie) [inline CHANGELOG.md]: This should be `5.0.0-alpha.14`.

**Comment by meenie** [comment]
@pkozlowski-opensource: Actually, I messed up.  `rxjs@5.0.0-alpha.14` didn't come in until `angular2@2.0.0-alpha.50`.  This is a changelog fix for 49 so it should be `rxjs@5.0.0-alpha.13`.

**Comment by IgorMinar** [comment]
I recreated this PR because it failed to rebase

**Pull request closed without merging**